nixos/tests/all-tests.nix

```diff
@@ -273,6 +273,7 @@ in {
   krb5 = discoverTests (import ./krb5 {});
   ksm = handleTest ./ksm.nix {};
   kubernetes = handleTestOn ["x86_64-linux"] ./kubernetes {};
+  ladybird = handleTest ./ladybird.nix {};
```

Comment:
should this be `handleTestOn` or is `platforms` in the ladybird package enough?

Reply:
I think handleTestOn is when not all platforms are supported